tesseract_ros/tesseract_examples/tesseract_ros_examples/src/pick_and_place_plan.cpp

```diff
@@ -234,7 +244,7 @@ int main(int argc, char** argv)
   pci.basic_info.dt_lower_lim = 2;    // 1/most time
   pci.basic_info.dt_upper_lim = 100;  // 1/least time
   pci.basic_info.start_fixed = true;
-  pci.basic_info.use_time = true;
+  pci.basic_info.use_time = false;
```

Comment:
start_fixed is true. At one point I thought we were updating the environment such that the start position was not all 0's. Right now it is straight up through the ceiling.

Reply:
Here's the positions we had.
```
  // Set the initial state of the robot
  std::unordered_map<std::string, double> joint_states;
  joint_states["iiwa_joint_1"] = 0.0;
  joint_states["iiwa_joint_2"] = 0.0;
  joint_states["iiwa_joint_3"] = 0.0;
  joint_states["iiwa_joint_4"] = -1.57;
  joint_states["iiwa_joint_5"] = 0.0;
  joint_states["iiwa_joint_6"] = 0.0;
  joint_states["iiwa_joint_7"] = 0.0;
  env->setState(joint_states);
```

Reply:
Found a bug. I added a unit test to catch if this ever occurs in tesseract_environment.